modules/checking-mco-node-status.adoc

```diff
@@ -13,6 +13,12 @@ To see the status of the Machine Config Operator (MCO) updates to your cluster,
 :FeatureName: Improved MCO state reporting
 include::snippets/technology-preview.adoc[]
 
+.Prerequisites
+* `TechPreviewNoUpgrade` feature set is enabled on your cluster.
+
+.Additional resources
+* See xref:../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling feature sets using the web console] for more information about feature gates.
```

Comment:
This section needs to be in the assembly after this module. We do not place xrefs in modules.